app/assets/stylesheets/govuk_publishing_components/components/govspeak/_steps.scss

```diff
@@ -11,13 +11,32 @@
       list-style-type: decimal;
```

Comment:
I think you can safely remove the first three lines here as they no longer apply?

```
background-position: 0 .87em;
background-repeat: no-repeat;
list-style-type: decimal;
```

Reply:
Maybe keep the list style type, although I'm not sure what difference it makes.

Reply:
Ah yes, I've removed background-position and background-repeat they're just things I missed.

I tried to do some history tracing of `list-style-type: decimal;` as it seemed a bit fishy (it relies on [overflow: hidden](https://github.com/alphagov/govuk_publishing_components/blob/5e682ed57f88cbedef61cafc0e62e4707170d38c/app/assets/stylesheets/govuk_publishing_components/components/govspeak/_steps.scss#L6) to hide them). I did some digging about the original approach but hit a dead end at https://github.com/alphagov/static/commit/1f41855b9e89690a3d5085e42bc0eef715a50801.

I wondered if it might be a way of still allowing numbers if someone disabled images, but that doesn't work (since they're still hidden by the overflow).

I've changed it to `list-style-type: none` since we've got no information recorded on why something quirky was done and it seems hard to imagine a case for it.